### PR TITLE
Send output when no gifs match tags.

### LINKED ISSUE #5: output when no results returned
hubot-slack adapter chokes when a message with no content is sent, and seems that this happens when there's no results:

```
   192	2016-02-13T01:41:55.548847+00:00 app[web.1]: [Sat Feb 13 2016 01:41:55 GMT+0000 (UTC)] ERROR TypeError: Cannot read property 'length' of undefined
   193	2016-02-13T01:41:55.548851+00:00 app[web.1]:   at SlackBot.send (/app/node_modules/hubot-slack/src/slack.coffee:226:7, <js>:275:16)
   194	2016-02-13T01:41:55.548852+00:00 app[web.1]:   at runAdapterSend (/app/node_modules/hubot/src/response.coffee:82:34, <js>:87:50)
   195	2016-02-13T01:41:55.548853+00:00 app[web.1]:   at allDone (/app/node_modules/hubot/src/middleware.coffee:44:37, <js>:34:16)
   196	2016-02-13T01:41:55.548854+00:00 app[web.1]:   at /app/node_modules/async/lib/async.js:274:13
   197	2016-02-13T01:41:55.548854+00:00 app[web.1]:   at Object.async.eachSeries (/app/node_modules/async/lib/async.js:142:20)
   198	2016-02-13T01:41:55.548855+00:00 app[web.1]:   at Object.async.reduce (/app/node_modules/async/lib/async.js:268:15)
   199	2016-02-13T01:41:55.548856+00:00 app[web.1]:   at /app/node_modules/hubot/src/middleware.coffee:49:7, <js>:37:22
   200	2016-02-13T01:41:55.548857+00:00 app[web.1]:   at nextTickCallbackWith0Args (node.js:453:9)
   201	2016-02-13T01:41:55.548857+00:00 app[web.1]:   at process._tickCallback (node.js:382:13)
```

Would be great to catch undefined results before kicking them to `robot.send` :)

### PR DESCRIPTION
**Pull request by patcon**
Resolves #5

### CODE REVIEW

**Comment by patcon** [comment]
also, output is good, because no response can be confused for hubot being down -- ours had been down for awhile, so I kept troubleshooting another 10 minutes when `hubot giphy all-night rager` resulted in silence :)

File: src/giphy-gifme.coffee [hunk 1]
```diff
@@ -17,6 +17,8 @@
 api_key = process.env.HUBOT_GIPHY_API_KEY or 'dc6zaTOxFJmzC' # <== Giphy's public API key, please request your own!
 rating_limit = process.env.HUBOT_GIPHY_RATING or 'pg'
 
+Array::empty = -> @.length == 0
```
Comment (bencentra) [inline src/giphy-gifme.coffee]: I suppose it doesn't matter in such a small script, but altering a built-in object's prototype is a bad practice.

I need to do some refactoring to get https://github.com/bencentra/hubot-giphy-gifme/pull/3 up to date and merged in, so I'll clean it up then.

**Comment by bencentra** [comment]
Despite the comment, :+1:

File: src/giphy-gifme.coffee (hunk 1 above)
Comment (patcon) [inline src/giphy-gifme.coffee]: heh that's totally fair. I'm clearly a rubyist trying to make myself at home :)

**Comment by bencentra** [comment]
Released as v0.1.6

**Comment by patcon** [comment]
:cat: ACK meow.

**Pull request merged**